ably/proto_protocol_message_test.go

```diff
@@ -24,7 +24,7 @@ func TestProtocolMessageEncodeZeroSerials(t *testing.T) {
 	encoded, err := ablyutil.MarshalMsgpack(msg)
 	assert.NoError(t, err)
 	// expect a 3-element map with both the serial fields set to zero
-	expected := []byte("\x83\xB0id\xA4test\xA9msgSerial\x00")
+	expected := []byte("\x82\xa2id\xa4test\xa9msgSerial\x00")
```

Comment:
The comment is now incorrect, this is a two-element map.

Reply:
updated